packages/frontend/src/creation-form/components/deploy/index.tsx

```diff
@@ -83,65 +85,36 @@ export const Deploy = ({
 
     const handleCreate = useCallback(() => {
         if (!tokenData) return;
-        const uploadToIpfsAndDone = async () => {
+        const create = async () => {
             setLoading(true);
-            let cid;
             try {
-                cid = await uploadToDecentralizeStorage(
+                assertRequiredOraclesData(oraclesData);
+                // TODO: implement the actual deployment
```

Comment:
What do you mean with the `TODO`?

Reply:
With this change the deployment is actually performed by template, while previously the template passed tx data to the host frontend and the frontend took care of the deployment. This makes it easier to handle the success step on the frontend of the template. Note we still have a way of notifying the host frontend that a deployment happened with the `onCreate` callback.

The TODO is because we need to actually handle the deployment on the template now (the logic is not there just yet).